azerothcore/azerothcore-wotlk
# fix(DB/Creature): Add auras to Jadefire Shadowstalker

Since the last PR i made about this topis a lot of info were passed into me. Because the branch was deleted a bit ago im redoing the PR with the data ive found on classic sniffs.

## Changes Proposed:
-  Deleted the smartAI events that casted sneak and slowing poison
-  Added those to the auras according to the sniffs. Those auras are [Faded](https://tbc.wowhead.com/spell=6408/faded) and [slowing poison](https://tbc.wowhead.com/spell=8601/slowing-poison).

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7056
- Closes https://github.com/chromiecraft/chromiecraft/issues/1226

## SOURCE:

Sniffs

## Tests Performed:

- .go c id 7110 
- Check the fade skill and while attaking him check that he have a 15% chance to apply slowing posion


## How to Test the Changes:


1.  .go c id 7110 
2. Check the fade skill and while attaking him check that he have a 15% chance to apply slowing posion



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
